app/main.js

```diff
@@ -176,21 +177,22 @@ const Timer = React.createClass({
     }
   },
   pause: function() {
+    currentTask.paused = true
```

Comment:
This is correct, but you need to find the `currentTask` first. So, just above this line, copy the line from line 172, i.e. `const currentTask = find(this.state.tasksList, {current: true})`

Then, when we assign `currentTask.paused = true`, the `currentTask` will be known.